pyproject.toml

```diff
@@ -15,9 +15,7 @@ authors = [{ name = "Anthony Scopatz" }, { email = "scopatz@gmail.com" }]
 maintainers = [{ name = "Xonsh Community" }, { email = "xonsh@gil.forsyth.dev" }]
 license = { text = "BSD 2-Clause License" }
 requires-python = ">=3.9"
-dependencies = [
-    "case-insensitive-dictionary; platform_system=='Windows'",
-]
+dependencies = ["requests; platform_system=='Windows'",]
```

Comment:
Could you just use the code directly? It would be a single class and the implementation is simple

Reply:
Yes, I like that better too.   Get rid of this dependency.....  I will do that and push.  I have to run for a plane.   I will try and push that at the airport.